### ISSUE TITLE
[YouVersion Bible Suggest] Chinese version Bible Parse Error

### ISSUE THREAD

**Issue by mikejhan4455** [raycast/extensions#7136]
### Extension

https://www.raycast.com/caleb531/youversion-suggest

### Description

The Chinese version of the bible seems to be causing a parse error, which is preventing any results or feedback from being displayed.

### Steps To Reproduce

1. Change the language settings to Traditional Chinese(繁體中文).
2. Select any version of whatever software application or tool is being used.
3. Even if "Look up ref" instruction is used, the results will still be in English instead of Chinese, despite the language setting being changed.
4. When using "Look up ref" instruction, there might not be any result displayed or returned.

### Current Behaviour

 The Chinese version of the bible causes a parse error and no results or feedback are displayed.

### Expected Behaviour

The Chinese version of the bible should be parsed without errors, allowing for results and feedback to be displayed.

**Comment by mikejhan4455** [comment]
Hi, thanks for the quickly reply!
The problem of "Search the Bible by Phrase" instruction is 

<img width="862" alt="Screenshot 2023-06-17 at 10 04 25 AM" src="https://github.com/raycast/extensions/assets/22511752/e05ef776-fd24-4514-aae2-d46c3c5148fc">


<img width="862" alt="Screenshot 2023-06-17 at 10 02 08 AM" src="https://github.com/raycast/extensions/assets/22511752/a941e759-8cd1-4621-8681-99f65126a72b">



<img width="716" alt="Screenshot 2023-06-17 at 10 02 51 AM" src="https://github.com/raycast/extensions/assets/22511752/21744d1d-f3f2-482d-81c2-
cde013557341">




<img width="862" alt="Screenshot 2023-06-17 at 10 01 44 AM" src="https://github.com/raycast/extensions/assets/22511752/1ad23288-9697-4de2-a9fb-28b12c506550">

**Comment by caleb531** [comment]
@mikejhan4455 A few things:

1. For the Search by Phrase issue, can you please send me the actual text you are using? I want to be able to copy/paste the Chinese characters so I can test with them (can't do that from a screenshot)
2. The extension has no knowledge of the alternative short names for each book, but I did confirm it will autocomplete the book name based on the first few characters entered (e.g. "馬太" matches ""馬太福音"). Now, I don't know Chinese, but I can understand if 馬太 is a completely different word, thus making searching this way potentially impractical (at least for Chinese).
3. I _can_ confirm that copied verse content is always coming back in English. I have found the root cause and have implemented a fix locally (which I will push in the next release)

**Comment by caleb531** [comment]
@mikejhan4455 Regarding the "Search by Phrase" issue, I was able to use my operating system's [Live Text](https://support.apple.com/guide/preview/interact-with-text-in-a-photo-prvw625a5b2c/mac) feature to grab the Chinese characters you were using. With that, I have been able to reproduce the issue and have fixed it locally.

<img width="774" alt="Screenshot 2023-06-17 at 11 06 15 AM" src="https://github.com/raycast/extensions/assets/872474/3204d873-edf5-4d71-a8dd-d3f8d1b1ea4f">

Regarding (2) from my previous comment, I don't plan to add support for shorthand book names since it would require substantial changes to the code.

With that, I plan to submit my other fixes today for review by the Raycast team. I expect it will be approved and published to the store within the next few days.
